uncontrast_studies/management/commands/load_uncon_data.py

```diff
@@ -27,15 +28,26 @@
 
 logger = logging.getLogger("UnConTrast")
 
-FILE_PATH = "uncontrast_studies/data/dataset_10052024.xlsx"
-ERROR_LOG_PATH = "uncontrast_studies/data/uncontrast_errors_log.xlsx"
+FILE_PATH = "uncontrast_studies/data/dataset_19052024.xlsx"
+MAOR_FILE_PATH = "uncontrast_studies/data/dataset_10052024.xlsx"
+FRANCOIS_FILE_PATH = "uncontrast_studies/data/dataset_10052024.xlsx"
+
+current_date = date.today()
+ERROR_LOG_PATH = f"uncontrast_studies/data/logs/uncontrast_errors_log_{current_date}.xlsx"
 
 
 class Command(BaseCommand):
     help = "Load uncon existing data"
 
     def handle(self, *args, **options):
         # Read .xlsx file and convert to dict
+        # maor_experiments_data = get_list_from_excel(MAOR_FILE_PATH, sheet_name="experiments")
```

Comment:
why is this comment here? for later?

Reply:
Francois wanted to work on a separate spreadsheet for the corrections process so I keep that option here as a comment until I get his files. it's temporary